index.html

```diff
@@ -2,13 +2,14 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>RDF/JS: Dataset specification</title>
+  <title>RDF/JS: Dataset specification 1.0</title>
   <script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove"></script>
   <script class="remove">
     var respecConfig = {
-      specStatus: "CG-DRAFT",
+      specStatus: "CG-FINAL",
+      publishDate: "2019-03-27",
       shortName: "rdfjs-dataset",
-      subtitle: "RDF/JS: Dataset specification",
+      subtitle: "RDF/JS: Dataset specification 1.0",
       processVersion: 2018,
```

Comment:
I would remove `processVersion`, it's been [removed](https://github.com/w3c/respec/blob/develop/CHANGELOG.md#v1950-2018-02-15).